libs/designer/src/lib/core/index.ts

```diff
@@ -74,3 +74,5 @@ export {
   getBrandColorFromManifest,
   getIconUriFromManifest,
 } from './utils/card';
+export { addOperation } from './actions/bjsworkflow/add';
+export { useAllOperations } from './queries/browse'
```

Comment:
Looks like linting has not been run on this file. (Missing `;` and newline at end of file.)

This would normally run as a pre-commit hook but it looks like maybe it didn't. You can run the following command from the repo root to address the linting in this file:

```
pnpm biome check --apply -- libs/designer/src/lib/core/index.ts
```